src/ripple/shamap/SHAMapItem.h

```diff
@@ -33,87 +34,58 @@ namespace ripple {
 class SHAMapItem
 {
 private:
-    uint256 mTag;
-    Serializer mData;
+    uint256    tag_;
```

Comment:
I take pride in code that is easy to read, even when it takes a few extra seconds to write it.